lapack/testlapack/dlarfg.go

```diff
@@ -56,6 +66,9 @@ func DlarfgTest(t *testing.T, impl Dlarfger) {
 				x[i] = rnd.Float64()
 			}
 		} else {
+			if len(test.x) != n-1 {
```

Comment:
Here is a reason why it would be nice to have precise vector lengths, but I agree that it's a difficult problem.